client/mobile-banner/index.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useState } from '@wordpress/element';
+import { Button, Icon } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { recordEvent } from '@woocommerce/tracks';
+
+/**
+ * Internal dependencies
+ */
+import { platform, ANDROID_PLATFORM } from '../lib/platform';
+import { AppIcon } from './app-icon';
+import './style.scss';
+import { PLAY_STORE_LINK, TRACKING_EVENT_NAME } from './constants';
+
+const SHOW_APP_BANNER_MODIFIER_CLASS = 'woocommerce-layout__show-app-banner';
+
+export const MobileAppBanner = ( { onInstall, onDismiss } ) => {
+	useEffect( () => {
+		const layout = document.getElementsByClassName(
+			'woocommerce-layout'
+		)[ 0 ];
+
+		if ( platform() === ANDROID_PLATFORM ) {
```

Comment:
I've tried to make this component as reusable as possible so we can still extract it for others to use such as wc.com. Although this logic is specific to us, it won't do anything if the `woocommerce-layout` element is not present. And as mentioned in the comments it may be able to go away when we refactor activity panel.